pyfuncol/tests/test_forbiddenfruit.py

```diff
@@ -0,0 +1,7 @@
+import pyfuncol
```

Comment:
I get why the other test file exists, but this one is not very useful since in the other tests we do exactly the same.

Reply:
Done.